pkg/dashboard/router.go

```diff
@@ -75,7 +76,7 @@ func GetRouter(setters ...Option) *mux.Router {
 			klog.Infof("404: %s", r.URL.Path)
 			http.NotFound(w, r)
 			return
-		}
+			}
```

Comment:
Can you please run a `go fmt ./...` ?

Reply:
Whoops! Sorry about that, I missed that. I've addressed it in ccefc44